src/app/api/application/uniform/route.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  createApplication,
+  updateApplication,
+  checkExistingReference,
+} from "@/app/actions/sanityClient";
+import { verifyApiKey } from "../../../lib/apiKey";
+import { NextRequest, NextResponse } from "next/server";
+import { validateUniformData } from "@/app/actions/uniformValidator";
+
+interface ApplicationError {
+  application: any;
+  error: string;
+}
+
+interface ApplicationResult {
+  success: string[];
+  errors: ApplicationError[];
+}
+
+/**
+ * @swagger
```

Comment:
Would be good if we could add the expected response to the swagger
eg for multiple 
```
{
    "data": {
        "success": [
            "Applciation 25553/5377/HSE created",
            "Applciation 26663/5377/HSE2 created"
        ],
        "errors": []
    }
}
```